modular/gater/object_handler.go

```diff
@@ -680,8 +680,12 @@ func (g *GateModular) queryUploadProgressHandler(w http.ResponseWriter, r *http.
 		taskState, errDescription, err = g.baseApp.GfSpClient().GetUploadObjectState(reqCtx.Context(), objectInfo.Id.Uint64())
 		if err != nil {
 			log.CtxErrorw(reqCtx.Context(), "failed to get uploading job state", "error", err)
+			if !strings.Contains(err.Error(), "no uploading record") {
```

Comment:
It is best to use some common functions for error handling such as non-existent/duplicate keys.

Reply:
If use a common function, there are many check conditions. I think write it in each function is ok.